src/components/header.js

```diff
@@ -1,92 +1,165 @@
 import React from 'react'
 import Link from './link'
 import Subbutton from './subbutton'
-import { css } from '@emotion/core'
+import {css} from '@emotion/core'
 import Container from './container'
-import { bpMinMD, bpMinSM } from '../utils/breakpoints'
+import {bpMinMD, bpMinSM, bpMaxMD} from '../utils/breakpoints'
+import VisuallyHidden from '@reach/visually-hidden'
+import Hamburger from './hamburger'
 
-const Header = ({ background, scrollToFooter }) => (
-  <header
-    css={css({
-      background: background,
-
-      [bpMinSM]: {
-        paddingTop: '40px',
-      },
-      paddingTop: '10px',
-      a: {
-        fontFamily: 'brandon-grotesque, sans-serif',
-        fontSize: '14px',
-        letterSpacing: '0.1em',
-        fontWeight: '700',
-        textTransform: 'uppercase',
-        margin: 0,
-      },
-    })}>
-    <Container
-      noVerticalPadding
+const Header = ({background, scrollToFooter}) => {
+  const [menuOpen, setMenuOpen] = React.useState(false)
+  const toggleMenu = () => setMenuOpen(value => !value)
+  return (
+    <header
       css={css({
-        display: 'flex',
-        alignItems: 'center',
-        justifyContent: 'space-between',
-      })}>
-      <h1>
-        <Link
-          css={css({
-            borderRadius: '4px',
-            color: 'rgb(79, 88, 95)',
-            [bpMinSM]: {
-              border: '2px solid #FFFFFF',
-              padding: '15px 20px',
-            },
-            padding: '0px',
-            fontWeight: '700',
-          })}
-          to='/'>
-          Illustrated.dev
-        </Link>
-      </h1>
+        background: background,
 
-      {/* Nav Grid Layout */}
-      <div
+        [bpMinSM]: {
+          paddingTop: '40px',
+        },
+        paddingTop: '10px',
+        a: {
+          fontFamily: 'brandon-grotesque, sans-serif',
+          fontSize: '14px',
+          letterSpacing: '0.1em',
+          fontWeight: '700',
+          textTransform: 'uppercase',
+          margin: 0,
+        },
+      })}
+    >
+      <Container
+        noVerticalPadding
         css={css({
-          [bpMinMD]: { display: 'flex' },
-        })}>
-        <Link
-          to='/'
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'space-between',
+        })}
+      >
+        <h1>
+          <Link
+            css={css({
+              borderRadius: '4px',
+              color: 'rgb(79, 88, 95)',
+              [bpMinSM]: {
+                border: '2px solid #FFFFFF',
+                padding: '15px 20px',
+              },
+              padding: '0px',
+              fontWeight: '700',
+            })}
+            to="/"
+          >
+            Illustrated.dev
+          </Link>
+        </h1>
+
+        {/* Nav Grid Layout */}
+        <div
```

Comment:
@ijones16 I think this should be a `<button>`, which would mean you wouldn't need a keyCode event. And then I think it'd be a good thing to use `aria-haspopup` along with `aria-expanded`. Nicely described [here](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html). ✌️